pkg/chunk/local/fs_object_client.go

```diff
@@ -70,3 +76,25 @@ func (f *fsObjectClient) getChunk(_ context.Context, decodeContext *chunk.Decode
 
 	return c, nil
 }
+
+// NewBucketClient makes a chunk.BucketClient which stores chunks as files in the local filesystem.
+func NewBucketClient(cfg FSConfig) (chunk.BucketClient, error) {
+	bucketClient, err := NewFSObjectClient(cfg)
```

Comment:
Can we expose the `fsObjectClient` type, and then return that?  That way we don't need the cast (or the extra constructor).